contrib/python-py-cpuinfo/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "python-py-cpuinfo"
```

Comment:
added this for benchmark so you can rebase it out

Reply:
Okay will drop